examples/javaee/README.md

```diff
@@ -0,0 +1,158 @@
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/redis/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Java EE Application using WildFly and MySQL
+
+The following document describes the deployment of a Java EE application using [WildFly](http://wildfly.org) application server and MySQL database server on Kubernetes. The sample application source code is at: https://github.com/javaee-samples/javaee7-simple-sample.
+
+### Prerequisites
+
+This example assumes that you have a Kubernetes cluster installed and running, and that you have installed the ```kubectl``` command line tool somewhere in your path.  Please see the [getting started](../../docs/getting-started-guides/) for installation instructions for your platform.
+
+### Start MySQL Pod
+
+In Kubernetes a [_Pod_](../../docs/user-guide/pods.md) is the smallest deployable unit that can be created, scheduled, and managed. Its a collocated group of Docker containers that share an IP and storage volume.
+
+Here is the config for MySQL pod: [mysql-pod.yaml](mysql-pod.yaml)
+
+Create the MySQL pod as follows:
+
+```sh
+kubectl create -f examples/javaee/mysql-pod.yaml
+```
+
+Check status of the pod as:
+
+```sh
+kubectl.sh get -w po
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   0/1       Pending   0          4s
+NAME        READY     STATUS    RESTARTS   AGE
+mysql-pod   0/1       Running   0          44s
+mysql-pod   1/1       Running   0         44s
+```
+
+Wait for the status to `1/1` and `Running`.
+
+### Start MySQL Service
+
+In Kubernetes a [_Service_](../../docs/user-guide/services.md) describes a set of Pods that perform the same task. An important use for a Service is to create a load balancer which distributes traffic across members of the set.  But a Service can also be used as a standing query which makes a dynamically changing set of Pods (or the single Pod we've already created) available via the Kubernetes API.
+
+The order of Service and the targeted Pods does not matter. However Service needs to be started before any other Pods consuming the Service are started.
```

Comment:
This explination of service is confusing. We should just link to the service doc and not go into depth about what a service is. Maybe say "We are creating a service (link to service doc) to expose the tcp port of the my sql server"